### ISSUE TITLE
Shall we retire this repo?

### ISSUE THREAD

**Issue by nothingismagick** [tauri-apps/tauri-gitian#2]
@kayabaNerve - you started the initiative, what are your thoughts about it?

**Comment by kayabaNerve** [comment]
Gitian is still the best option for reproducible builds.

The reason I stopped working on this is because when I worked on this months ago, I couldn't get Gitian to use the needed environment variables for some reason. My plan was to complete Tauri's Gitian setup when I did one for my own work. The one for my own work has yet to happen though, so neither has Tauri's.

I'll try again to get a working setup within the next week. If I get a working setup, I'll complete the scripts for Tauri. If I can't, we might as well for now.
